client/rest-high-level/src/test/java/org/opensearch/client/core/tasks/GetTaskResponseTests.java

```diff
@@ -38,12 +38,12 @@
 import org.opensearch.core.xcontent.ToXContent;
 import org.opensearch.core.xcontent.XContentBuilder;
 import org.opensearch.tasks.RawTaskStatus;
-import org.opensearch.tasks.TaskResourceStats;
-import org.opensearch.tasks.TaskResourceUsage;
+import org.opensearch.core.tasks.resourcetracker.TaskResourceStats;
+import org.opensearch.core.tasks.resourcetracker.TaskResourceUsage;
 import org.opensearch.tasks.Task;
-import org.opensearch.tasks.TaskId;
+import org.opensearch.core.tasks.TaskId;
```

Comment:
Super nit (may benefit from package ordering, I was under impression `spotless` does it)

Reply:
I wish it did. I'm trying to lazily address these without putting too much into it since imports are changing so often.